src/js/events/MetronomeActions.js

```diff
@@ -74,10 +65,21 @@ const MetronomeActions = {
               });
               resolve();
             } catch (error) {
-              handleError(error);
+              AppDispatcher.handleServerAction({
+                type: REQUEST_METRONOME_JOBS_ERROR,
+                data: error
+              });
+              reject();
             }
           },
-          error: handleError,
+          error(xhr) {
+            AppDispatcher.handleServerAction({
+              type: REQUEST_METRONOME_JOBS_ERROR,
+              data: xhr.message,
```

Comment:
@MatApple @nLight Let's invest the time to properly clean things up as we don't have a schedule for the GraphQL refactoring and I don't see an urgent need for merging this PR.